src/components/Navbar/Drawer/index.tsx

```diff
@@ -94,13 +97,17 @@ const Drawer: React.FC<Props> = ({
 
   const closeDrawer = useCallback(
     (actionSource = 'click') => {
+      if ((actionSource === 'esc_key' || actionSource === 'outside_click') && !closeOnBlur) {
```

Comment:
1. why do we need to check for those 2 action sources only? there is a third one `navigation`.

2. We should add an ActionSource enum and use it here instead of strings